### PR TITLE
uploading version 1

### PR DESCRIPTION
**Pull request by anastasyadybchenko**
Hi, i have created the first version of Carbón Parrillero

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        
+        
+      <link rel="stylesheet" href="styles.css?16">
+    </head>
+    
+    <body>
+      <div id="bg_home" class="img-responsive backgrounds">
+        <div id="home" class="sections">
```
Comment (jorgecarrarav) [inline src/index.html:16]: Shouldn't it be section_home?

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                              </a>
+                        </div>
+                        <h2 id="product6_title" onclick="productOnClick(6)">desk6</h2>
+                        <h4 id="product6_description" onclick="productOnClick(6)">Probably some describtions here6</h4>
+                    </div>
+                </div>
+           </div>
+         </div>
+        </div> 
+            <div id="fixed" class="backgrounds">
```
Comment (jorgecarrarav) [inline src/index.html:102]: what is the meaning of Fixed? should it be something like "section_3"  ?

Comment (anastasyadybchenko) [reply src/index.html:102]: This is how the tech. leader told me to do it this time

Comment (jorgecarrarav) [reply src/index.html:102]: ok

**Pull request merged**